features/FEATURE_PSA/TARGET_TFM/src/os_wrapper_cmsis_rtos_v2.c

```diff
@@ -11,31 +11,60 @@
 
 #include <string.h>
 #include "cmsis_os2.h"
+#include "rtx_os.h"
+#include "stdlib.h"
 
 /* This is an example OS abstraction layer for CMSIS-RTOSv2 */
 
 void *os_wrapper_thread_new(const char *name, int32_t stack_size,
                             os_wrapper_thread_func func, void *arg,
                             uint32_t priority)
 {
-    osThreadAttr_t task_attribs = {.tz_module = 1};
+    osRtxThread_t *thread;
+    void *stack;
 
-    /* By default, the thread starts as osThreadDetached */
-    if (stack_size != OS_WRAPPER_DEFAULT_STACK_SIZE) {
-        task_attribs.stack_size = stack_size;
+    thread = (osRtxThread_t *)malloc(osRtxThreadCbSize);
+    if (thread == NULL) {
+        return NULL;
     }
-    task_attribs.name = name;
-    task_attribs.priority = (osPriority_t) priority;
 
-    return (void *)osThreadNew(func, arg, &task_attribs);
+    stack = (void *)malloc(stack_size);
```

Comment:
why is the check for `stack_size != OS_WRAPPER_DEFAULT_STACK_SIZE` removed? The value of `OS_WRAPPER_DEFAULT_STACK_SIZE` is -1. So if tests wants to use default stack size then -1 will be passed as argument. `malloc` will fail in that case.

Reply:
When we are managing the memory manually, we don't have a default size of the stack which can be allocated by the OS. There it would be good to force the developer to specify the size required.

Reply:
> Patch - [84c3916](https://github.com/ARMmbed/mbed-os/commit/84c39161ca39fbf8298032038b1f05ea50f41a7d) is going to be upgraded with new Mbed OS C++ implementation as requested by the reviewer. On a standalone basis, this commit works, should not cause any harm.
> 
> Patch - [d1ff232](https://github.com/ARMmbed/mbed-os/commit/d1ff232a5d389badcd55df409fc45901b0aeeb91) is required for `TARGET_CY8CPROTO_064_SB` since this a prototype and development is been halted. Seeing this target has played an important role in bring-up of TF-M + Mbed-os integration, therefore this patch has been added.

The changes done in the commit https://github.com/ARMmbed/mbed-os/tree/ad7014054b7974b48a63bef2c2066b93714da2e3 are removed by https://github.com/ARMmbed/mbed-os/tree/daf9fddbb2fb7f81e3944a2ec26684014db90b47. So why do we want to keep previous one?

Reply:
Because this is a design change, On a standalone basis this commit will ok.